### PR TITLE
Batch 2 of tpch queries for polars and pandas

### PR DESCRIPTION
**Pull request by chitralverma**
- Added code for q2 polars & pandas
 - Added code for q3 pandas
 - Added code for q4 polars & pandas
 - Added code for q5 polars & pandas

### CODE REVIEW

File: polars_queries/q4.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from linetimer import linetimer
+
+from polars_queries import polars_tpch_utils
+
+Q_NUM = 4
+
+
+@linetimer(name=f"Overall execution of Query {Q_NUM}", unit='s')
+def q():
+    var1 = "1993-07-01"
```
Comment (ritchie46) [inline polars_queries/q4.py:12]: Can we use python datetime here

File: polars_queries/q5.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+from linetimer import linetimer
+
+from polars_queries import polars_tpch_utils
+
+Q_NUM = 5
+
+
+@linetimer(name=f"Overall execution of Query {Q_NUM}", unit='s')
+def q():
+    var1 = "ASIA"
```
Comment (ritchie46) [inline polars_queries/q5.py:12]: Python dataetime

**Comment by ritchie46** [comment]
Thanks, I think it looks good. Can I merge it?

**Comment by chitralverma** [comment]
let me do q6 as well,

**Comment by ritchie46** [comment]
> let me do q6 as well,

Can we do that in a separate PR?

**Comment by chitralverma** [comment]
> > let me do q6 as well,
> 
> Can we do that in a separate PR?

done already
 :D

**Comment by chitralverma** [comment]
Added code for q2 and q6 as well.

**Comment by ritchie46** [comment]
Thanks a lot for the effort @chitralverma!

**Pull request merged**